docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1096 @@
+        }
+    }
+    ```
+
+
+-   Set the specific engine result callback invoked when voice control engine sends additional information about recognition result not the registered commands:
+
+    ```csharp
+    /// callback
+    void specificEngineResult(object sender, SpecificEngineResultEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetSpecificEngineResultCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.SpecificEngineResult += specificEngineResult;
+
+            /// Unset the callback
+            VoiceControlManagerClient.SpecificEngineResult -= specificEngineResult;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the pre recognition result updated callback invoked when voice control engine sets the pre recognition results (partial ASR) to voice control manager.
+
+    You can get the results of pre result event type and the pre recognition results, the partial ASR.
+
+    ```csharp
+    /// callback
+    void preRecognitionResultUpdated(object sender, PreRecognitionResultUpdatedEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetPreRecognitionResultUpdatedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.PreRecognitionResultUpdated += preRecognitionResultUpdated;
+
+            /// Unset the callback
+            VoiceControlManagerClient.PreRecognitionResultUpdated -= preRecognitionResultUpdated;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the all recognition result received callback invoked when voice control engine sends the all recognition result to voice control manager.
+
+    In the callback, the recognized result, recognized text, and engine message results are included and recognized result can include more than two voice commands if two voice commands those have same command string are registered by each voice control clients.
+
+    If you want to select specific command in the recognized result, you can use SetRecognizedCommandsSelectionDelegate() method.
+
+    ```csharp
+    /// callback
+    void allRecognitionResultReceived(object sender, AllRecognitionResultEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetAllRecognitionResultReceivedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.AllRecognitionResultReceived += allRecognitionResultReceived;
+
+            /// Unset the callback
+            VoiceControlManagerClient.AllRecognitionResultReceived -= allRecognitionResultReceived;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the recognition result updated callback invoked when voice control engine set the recognition result updated to voice control manager.
```

Comment:
Question. How about below? Would you please check?

Set the recognition result updated callback invoked when voice control engine updates the recognition result to voice control manager.